examples/1-hello-world/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "hello-world",
+  "version": "0.0.1",
+  "description": "Hello world example",
+  "main": "index.js",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/copilot-extensions"
+  },
+  "scripts": {
+    "start": "node index.js",
+    "watch": "node --watch index.js"
```

Comment:
does node support a `--watch` flag now?

Reply:
Yep

Reply:
woah today I learned, thanks! 
https://nodejs.org/api/cli.html#--watch

Reply:
Copilot told me it was available!